app/components/ui/atoms/CalloutBox.js

```diff
@@ -2,16 +2,36 @@ import React from 'react'
 import styled from 'styled-components'
 import { Box } from 'grid-styled'
 import { th } from '@pubsweet/ui-toolkit'
+import Icon from './Icon'
 
-const CalloutBox = styled(({ enabled, ...rest }) => <Box {...rest} />).attrs({
-  mx: -3,
-  px: 3,
-  mb: 3,
+const Frame = styled(props => <Box {...props} />).attrs({
+  mb: 2,
 })`
   border: ${th('borderWidth')} ${th('borderStyle')} ${th('borderColor')};
   border-radius: ${th('borderRadius')};
-  border-color: ${({ enabled = true }) =>
-    enabled ? th('colorBorder') : 'transparent'};
+  border-color: ${th('colorBorder')};
+  position: relative;
 `
 
+const CloseButton = styled(Icon)`
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  padding: ${th('space.2')};
+  svg {
+    stroke: ${th('colorFurniture')};
+    fill: ${th('colorFurniture')};
+  }
+`
+
+const CalloutBox = ({ closable, onClose, children, ...props }) => (
```

Comment:
Would it be better to have `onClose` implicitly make the box closable? Or 
 are there situations where you could use one prop and not the other?

Reply:
Possibly. I thought it might be a bit magical. Thoughts one way or the other?